src/components/Steps/__stories__/steps.stories.mdx

```diff
@@ -0,0 +1,130 @@
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Steppers display progress through a sequence of logical and numbered steps. They may also be used for navigation.
+<Canvas>
+  <Story name="Overview"
+         args={ {steps: steps5, activeStepIndex: 2} }>
+    { stepsTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Steps } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Stepper must have three or more steps.",
+  "The stepper shows users where they are in the process, and can be used to navigate through the process by selecting steps.",
+  "If a task needs more than six steps, consider simplifying the process or breaking it up into multiple tasks.",
+]}/>
+
+## Variants
+### Types
+Steps with a number view or gallery view.
+<Canvas>
+  <Story name="Types">
+    <div className="monday-style-story-steps_column-wrapper">
+      <Steps type={Steps.types.NUMBERS} steps={steps6} activeStepIndex={4} />
+      <Steps steps={steps5} activeStepIndex={2} />
+      <Steps steps={steps5} activeStepIndex={2} areNavigationButtonsHidden className="monday-style-story-steps_padding" />
+    </div>
+  </Story>
+</Canvas>
+
+### On primary
+<Canvas>
+  <Story name="On primary">
+    <div className="monday-style-story-steps_column-wrapper monday-style-story-steps_color" >
+      <Steps steps={steps6} activeStepIndex={4} isOnPrimary type={Steps.types.NUMBERS} />
+      <Steps steps={steps5} activeStepIndex={2} isOnPrimary />
+      <Steps steps={steps5} activeStepIndex={2} isOnPrimary areNavigationButtonsHidden className="monday-style-story-steps_padding" />
+    </div>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:<Steps type={Steps.types.NUMBERS} steps={steps6} activeStepIndex={4} />,
+        description:"Use steps with numbers type for use cases of steps with more than 5 steps."
+      },
+      negative: {
+        component:<Steps steps={steps6} activeStepIndex={3} />,
+        description:"Don't use the gallery type steps component for more than 5 steps."
+      }
+    }
+  ]}
+/>
+
+## Use cases and examples
+### Steps inside a tipseen
+Our Tipseen component includes support for steps as content.
+<Canvas>
+  <Story name="Steps inside a tipseen">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <div className="monday-style-story-steps_block">
+          <Tipseen 
+            position={Tipseen.positions.LEFT} 
+            modifiers={modifiers}
```

Comment:
I think there is no need for this helper in the steps stories because the jump problem is not appear there

Reply:
Unfortunately, without it, the Tipseen would be jumping

Reply:
If we remove it, the Tipseen in the use case will  jump